packages/frontend-api/src/Model/Resolver/Advert/AdvertsQuery.php

```diff
@@ -43,10 +43,6 @@ public function advertsQuery(?string $positionName = null, ?string $categoryUuid
 
         $category = null;
 
-        if ($categoryUuid !== null) {
```

Comment:
this can stay... otherwise we drop some functionality from frontend-api package.